CHANGELOG.md

```diff
@@ -6,6 +6,39 @@ and this project uses `yyyy.rr[.pp]`, where `yyyy` is the year a patch is releas
 `rr` is a sequential release number (starting from `01`), and an optional two-digit
 sequential patch number (starting from `01`).
 
+## [2024.02] - 2024-07-?
+
+### Added
+- SIMPLE: Adds option to obtain concurrent drag and heat transfer coefficents via `monin_obukhov` functions. To control the new option, new namelist options have been added to the `surface_flux_nml`. The `rough_scheme_ocean` in `ocean_rough_nml` must be set to `hwrf17` for the options to be effective. (#113)
+	- `do_iter_monin_obukhov` controls whether to iterate over ocean with updated roughness length
+	- `niter_monin_obukhov` controls the number of times to iterate
+	- `use_u10_neutral` if true uses 10m neutral wind rather than the standard 10m wind to obtain `rough_mom`, `rough_heat`, and `rough_moist`
+
+### Changed
+- FULL: A number of changes have been made as part of a refactorization effort for the full coupler.
+	- A module `full_coupler_mod` has been added and contains routines and imports previously included in the `coupler_main.F90` file. (#104)
+	- A new type, `coupler_clock type`, has been added to handle the usage of any clocks used in the full coupler. This will cause changes in the output order of clock statistics (#106)
+	- Updates the setting of version number to be accurate (#109)
```

Comment:
I'm not sure about the accurate part, pehraps Updates the setting of version number to use file_version.fh